ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -77,7 +80,8 @@ class PostBindingVerifier : BindingVerifier() {
      * 4.1.4.5 POST-Specific Processing Rules
      */
     fun verifyPostSSO(samlResponseDom: Node) {
-        if (samlResponseDom.children(SIGNATURE).isEmpty()
+        if (!samlResponseDom.nodeName.contains("LogoutResponse")
```

Comment:
✏️ Does it make more sense to change `!samlResponseDom.nodeName.contains("LogoutResponse")` to `samlResponseDom.nodeName.contains("Response")`?

Reply:
`LogoutResponse` also contains `Response`

Reply:
Whoops!

Reply:
I think what @bakejeyner said makes more sense though. This line is going to have to be changed if we do anything other than SSO or SLO. Do we have to do contains because the `nodeName` includes the namespace, if present? Could we just trim the namespace off and do a `==` instead?